Source/WebKit/WebProcess/WebPage/EventDispatcher.cpp

```diff
@@ -342,4 +335,26 @@ void EventDispatcher::pageScreenDidChange(PageIdentifier pageID, PlatformDisplay
 #endif
 }
 
+#if ENABLE(MOMENTUM_EVENT_DISPATCHER)
+void EventDispatcher::setScrollingAccelerationCurve(PageIdentifier pageID, std::optional<ScrollingAccelerationCurve> curve)
+{
+    m_momentumEventDispatcher->setScrollingAccelerationCurve(pageID, curve);
+}
+
+void EventDispatcher::handleSyntheticWheelEvent(WebCore::PageIdentifier pageIdentifier, const WebWheelEvent& event, WebCore::RectEdges<bool> rubberBandableEdges)
+{
+    internalWheelEvent(pageIdentifier, event, rubberBandableEdges, WheelEventOrigin::MomentumEventDispatcher);
+}
+
+#if ENABLE(MOMENTUM_EVENT_DISPATCHER_TEMPORARY_LOGGING)
+void EventDispatcher::flushMomentumEventLoggingSoon()
```

Comment:
Why does this have to be delayed?

Reply:
So that it didn't interfere with performance during the gesture. Note that this logging is off by default.